.drone.yml

```diff
@@ -25,17 +30,26 @@ pipeline:
       branch: master
       event: push
 
-  deploy_to_dev:
-    image: quay.io/ukhomeofficedigital/kd:v0.8.0
+  tagged_image_to_quay:
+    image: docker:17.09.0-ce
     secrets:
-      - kube_token
+      - docker_password
     environment:
-      - KUBE_SERVER=https://kube-dev.dsp.notprod.homeoffice.gov.uk
-      - KUBE_NAMESPACE=dev-induction
-      - INSECURE_SKIP_TLS_VERIFY=true
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - cd kube
-      - kd -f deployment.yml -f ingress.yml -f service.yml
+      - docker login -u="ukhomeofficedigital+acp_node_hello_world" -p=$${DOCKER_PASSWORD} quay.io
+      - docker tag acp-node-hello-world:$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/acp-node-hello-world:$${DRONE_TAG}
```

Comment:
You haven't named the image you built in the previous step - will this work?

Reply:
`acp-node-hello-world` -> `developer-induction`, and remove `:$${DRONE_COMMIT_SHA}`